x/slashing/handler.go

```diff
@@ -1,9 +1,30 @@
 package slashing
 
 import (
+	"strings"
+
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
+const (
+	SideChainIdBsc    int64 = 56
+	SideChainIdChapel int64 = 97
+	SideChainIdRialto int64 = 714
+)
+
+func SideChainIdFromText(str string) int64 {
+	switch strings.ToLower(str) {
+	case "bsc":
+		return SideChainIdBsc
+	case "chapel":
+		return SideChainIdChapel
+	case "rialto":
+		return SideChainIdRialto
+	default:
+		return 48
```

Comment:
After second thought, I think it is better to panic here. What do you think?

Reply:
Sure. If the chainID provided is not valid, it's meaningless to return a default chainID if it doesn't match.